harness/bin/test_harness_driver.py

```diff
@@ -200,7 +200,7 @@ def auto_generated_scripts(harness_config,
         run_count = 1
         max_count = 1
         max_subs_cfg = mymachine.test_config.get_max_submissions()
```

Comment:
get_max_submissions() returns either the value from config or None. These should be effectively the same.

Reply:
Actually this has different outcomes if an empty string is returned. I'm not sure an empty string can be pulled from the config file.

Reply:
Not enough coffee. The proposed change WOULD handle an empty string appropriately.

```
>>> val=""
>>> if val is None: print("true")
... 
>>> if not val: print("true")
... 
true
```